libnavui-androidauto/build.gradle

```diff
@@ -40,12 +40,11 @@ dependencies {
 
     // This defines the minimum version of Navigation which is included in this SDK. To upgrade the
     // Navigation versions, you can specify a newer version in your downstream build.gradle.
-    releaseApi("com.mapbox.navigation:android:2.9.0-rc.2")
-    debugApi project(":libnavigation-android")
+    api("com.mapbox.navigation:android:2.9.1")
 
     // Search is currently in beta so it is not included in this SDK. The functionality of search
     // is included behind this library's api.
-    implementation(dependenciesList.mapboxSearchSdk)
+    implementation("com.mapbox.search:mapbox-search-android:1.0.0-beta.38.1")
```

Comment:
I think this should be reverted. `1.0.0-beta.38.1` uses incompatible version of Common SDK. 
Also `mapboxSearchSdk` is not used anywhere else, we can safely update it, if necessary for Android Auto.

Reply:
The libnavui-androidauto sdk will be pinned to a separate version of search. Also bumping it causes a crash https://github.com/mapbox/mapbox-navigation-android/pull/6588#issuecomment-1312292125

Reply:
We can remove the constant then. Also it is really strange that you run into this crash. It doesn't happen in 1TAP, where we have been using that version for a long time.

Reply:
updated to use 39 again. that crash must be local to my machine. also 39 should be the [better version to use](https://github.com/mapbox/mapbox-search-android/compare/v1.0.0-beta.38.1...v1.0.0-beta.39#diff-6f145b4e527ee2ba6561f96e650878e03af25cfaf1a7f41c1dc580e61a43c9b7L44-L49)

Reply:
Cool thanks for testing too, this works for me after `rm -rf ~/.gradle/caches`